content/en-US/api/2dcontextrestoreevent.md

```diff
@@ -0,0 +1,38 @@
+---
+recipe: api-interface
+title: '2dcontextrestored'
+mdn_url: /en-US/docs/Web/API/2dcontextrestored_event
+specifications: [[specURL]]#[[HeadingAnchor]]
+browser_compatibility: api.2dcontextlost
+---
+
+**When this feature ships, the content below will live on MDN under
+[developer.mozilla.org/en-US/docs/Web/API](https://developer.mozilla.org/en-US/docs/Web/API).**
+
+## Description
+
+The 2d Rendering context losses may be intended by the user agent (to resolve
+resource contention), or may be forced by external factors (e.g. a graphics
+driver reset). If the context is lost then restored, this event will be triggered.  
```

Comment:
Same as above, I feel like you need to start this explanation from somewhere more basic.

Reply:
I change it so it's the same style as the context lost event.